tests/source/lhs-to-rhs-between-comments/assignment-multi-line-comments.rs

```diff
@@ -0,0 +1,81 @@
+// rustfmt-version: Two
+// "=" Assignemnt - Multiline Block comments
+fn main () {
+let var = /* Block comment line 1 
+* Block comment line 2
+* Block comment line 3 */ first;
+var = /* Block comment line 1 
+* Block comment line 2
+* Block comment line 3 */ second;
+var = /* Block comment line 1 longggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
+* Block comment line 2
+* Block comment line 3 */ third;
+var = /* Block comment line 1 
+* Block comment line 2 longggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
+* Block comment line 3 */ forth;
+}
+
+// "=" Assignemnt - Multiline Line and Block comments
+fn main () {
+let var = // Line comment 1
+// Line comment 2
+// Line comment 3
+first;
```

Comment:
I noticed that none of these tests are written in the form `let pat: ty = expr`. Could you include some tests that use type annotations.

Reply:
Added related test cases.